<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>396</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>起来站站</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="confirm_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="interval_la">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>间 隔 时 间(m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="notigy_la">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>提 醒 次 数(s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">关闭提醒</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>45</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="cd_la">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>倒 计 时 (s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remake_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>重新计时</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>循 环 开 关</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBu_open">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>开</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBu_close">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>提  示  语</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="hint_pt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
